lib/galaxy/jobs/runners/util/job_script/DEFAULT_JOB_FILE_TEMPLATE.sh

```diff
@@ -4,6 +4,10 @@
 # to ensure this script is runnable before running it directly
 # or submitting it to a cluster manager.
 if [ -n "$ABC_TEST_JOB_SCRIPT_INTEGRITY_XYZ" ]; then
+    $command
```

Comment:
Well this is terrible and awful, but the fact you reproduce this locally means at least we aren't just fixing this for jenkins.

I'll take a crack at reworking this and open a PR with even longer waits.